api/services/MessagingService.js

```diff
@@ -0,0 +1,13 @@
+import findOrCreateThread from '../models/post/findOrCreateThread'
+import createComment from '../models/comment/createComment'
+
+const MessagingService = {
+  async sendMessageFromAxolotl (userIds, text) {
+    // find existing message thread or create a new one
```

Comment:
this comment is kind of useless now -- i think i just put it there while we were writing this as a temporary reminder